### PR TITLE
OCLOMRS-333: Fix unresponsive button on bulk concepts page

### PR DESCRIPTION
**Pull request by Tittoh**
# JIRA TICKET NAME:
[Fix unresponsive button on bulk concepts page](https://issues.openmrs.org/browse/OCLOMRS-333)

# Summary:
This fixes the back button in the bulk concepts page. Currently, the button is sometimes unresponsive and is fixed by this PR.

### CODE REVIEW

**Comment by dkayiwa** [comment]
Is all this necessary for fixing the back button?
There is a lot of formatting changes which should be in their own ticket and pull request.

**Comment by Tittoh** [comment]
I have reverted the formatting caused by auto fixing linter warnings.

File: src/tests/bulkConcepts/addBulkConcepts.test.jsx [hunk 1]
```diff
@@ -28,6 +28,7 @@ describe('Add Bulk Concepts', () => {
       isFetching: true,
       match,
       addExistingBulkConcepts: jest.fn(),
+      language: 'en',
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/addBulkConcepts.test.jsx]: How is this related to the fix of the back button?

Comment (Tittoh) [reply src/tests/bulkConcepts/addBulkConcepts.test.jsx]: yes. the link has a `language` of `en` at the end.
e.g. `http://localhost:3000/concepts/users/admin/858738987555379/CHD/en/`
